base/uk.ac.stfc.isis.ibex.scriptgenerator/python_support/script_definition_loader.py

```diff
@@ -222,6 +240,23 @@ def estimateTime(self, action, global_params) -> Union[None, int]:
         except (ValueError, TypeError, KeyError) as ex:
             return None
 
+    def estimateCustom(self, action, global_params) -> Union[None, List[int]]:
```

Comment:
```
-> Union[None, List[int]]
```

We can't restrict ourselves to `int` outputs, in general this needs to be either `float` - or even more generally, `str`, and do the formatting of number -> str on the python side.